security/form_login_setup.rst

```diff
@@ -105,7 +105,7 @@ Edit the security.yml file in order to allow access to the ``/login``route ::
 
     .. code-block:: php
 
-        // config/packages/security.php
+        // app/config/security.php
```

Comment:
This cannot change when targeting 4.2, maybe you should target 3.4 instead.

Reply:
My bad on this one, a sad copy paste hehe Thank you !